web/viewer.css

```diff
@@ -135,32 +135,6 @@ select {
   cursor: default;
 }
 
-/* outer/inner center provides horizontal center */
-.outerCenter {
-  pointer-events: none;
```

Comment:
I checked and it shouldn't, but please check as well. I think what caused this before is https://github.com/mozilla/pdf.js/pull/7781/files#diff-5a36ed3216dce1e4da7e938ee4503ebdL157. Due to this rule, not the entire container (`outerCenter`), but only the inside (`innerCenter`) was moved, which caused it to be moved outside of the parent container. You can verify this by opening https://mozilla.github.io/pdf.js/web/viewer.html and inspecting the `outerCenter` and `innerCenter` elements. Notice that `outerCenter` is at a different position than `innerCenter`, while they should be the same. The new CSS no longer has this problem as `innerCenter` is gone.